packages/@aws-cdk/aws-elasticloadbalancingv2/test/integ.alb.stickiness.ts

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env node
```

Comment:
I'm not sure this feature deserves its own integ tests. I might just add one of the stickiness parameters to the existing integ test and call it a day.

Reply:
I'm channeling Werner Vogels  here - ‘Dance Like Nobody’s Watching. ~Encrypt~ Test Like Everyone Is’ 😄

Reply:
@njlynch consolidated